UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs

```diff
@@ -105,6 +105,8 @@ protected set
 	//can go up or down based on possible sources of being on fire. Max seems to be 20 in tg.
 	[SyncVar(hook=nameof(SyncFireStacks))]
 	private float fireStacks;
+	private float maxFireStacks = 20f;
```

Comment:
20 seems like too much in my opinion, I don't think it should be more than 5. Correct me if I'm wrong though (if /tg/ has 20 too then that's fine)

Reply:
When I asked in discord 20 was the number thrown around and referenced in our code comments, but I have no real opinion on this as anything over 5 seems to be fatal even with stop drop and rolls.

Reply:
sorry for late reply. make it 5

Reply:
dont sweat it, changed to 5